src/reflect/makefunc.go

```diff
@@ -0,0 +1,5 @@
+package reflect
+
+func MakeFunc(typ Type, fn func(args []Value) (results []Value)) Value {
+	return Value{}
```

Comment:
I do not think returning `Value{}` is correct here. For example, `reflect.MakeFunc(...).Kind()` would return `reflect.Invalid` instead of `reflect.Func`. Therefore, I think this should panic instead.

Reply:
Yeah, I wasn't sure about this. Thanks for pointing it out. I will update it.

Reply:
Pushed.